.env.example

```
@@ -17,4 +17,5 @@ NEXT_PUBLIC_KVM_RELEASE_DATE_TIMESTAMP_MS=
 NEXT_PUBLIC_ENABLE_ASSET_APPLY=true
 NEXT_PUBLIC_EXCHANGE_URL=
 NEXT_PUBLIC_BRIDGE_URL=
-NEXT_PUBLIC_SWAP_URL=
\ No newline at end of file
+NEXT_PUBLIC_SWAP_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SWAP_URL key should go before the NEXT_PUBLIC_TRANSFER_ADDRESS key

(UnorderedKey)